dbt/include/trino/macros/crossdb/datediff.sql

```diff
@@ -0,0 +1,28 @@
+{% macro trino__datediff(first_date, second_date, datepart) %}
```

Comment:
it would be good to have a usage sample of `datediff` in an integration test to showcase for newbies how these macros can get used.
WDYT?

Reply:
There is actually an integration test for this feature already. See https://github.com/starburstdata/dbt-trino/blob/66d50bbc863e15233cd8fd7a65cb7edf9b039fdd/tests/functional/adapter/test_utils.py#L46